staging/src/k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1/types.go

```diff
@@ -25,7 +25,10 @@ type CustomResourceDefinitionSpec struct {
 	// Group is the group this resource belongs in
 	Group string `json:"group" protobuf:"bytes,1,opt,name=group"`
 	// Version is the version this resource belongs in
-	Version string `json:"version" protobuf:"bytes,2,opt,name=version"`
+	// Should be always first item in Versions field if provided.
```

Comment:
add the deprecation

Reply:
I wish we had `// +deprecated`.  I added a line with `deprecated:` at the beginning according to godoc.